themes/bootstrap3/js/lightbox.js

```diff
@@ -250,10 +250,14 @@ VuFind.register('lightbox', function Lightbox() {
    */
   _constrainLink = function constrainLink(event) {
     var $link = $(this);
+    var urlRoot = location.origin + "/" + VuFind.path;
+
     if (typeof $link.data("lightboxIgnore") != "undefined"
       || typeof $link.attr("href") === "undefined"
       || $link.attr("href").charAt(0) === "#"
       || $link.attr("href").match(/^[a-zA-Z]+:[^/]/) // ignore resource identifiers (mailto:, tel:, etc.)
+      || ($link.attr("href").slice(0, 4) == "http" // external links
```

Comment:
Is there a reason to use `==` instead of `===` here? GitHub Actions is complaining about it.

Reply:
For string comparison there is no difference so == is usually clearer but that's not what our styles enforce - will fix.

Reply:
Done.